core/service/manager.go

```diff
@@ -169,13 +122,7 @@ func (manager *Manager) Kill() error {
 		err = manager.dialogWaiter.Stop()
 	}
 
-	if manager.natService != nil {
-		manager.natService.Stop()
-	}
-
-	if manager.vpnServer != nil {
-		manager.vpnServer.Stop()
-	}
+	manager.service.Stop()
```

Comment:
`manager.service.Stop()` returns an error too, should we check it?

Reply:
Added handling in any of subcomponent stop.